test/firtool/firtool.fir

```diff
@@ -40,3 +40,19 @@ circuit test_mod :
 ; MLIRLOWER: }
 
 ; MLIRLOWER: }
+
+  module implicitTrunc :
+    input inp_1: UInt<1>
+    input inp_2: SInt<5>
+    output out1: SInt<3>
+    output out2: SInt<3>
+    out1 <= dshl(inp_2, inp_1)
+    out2 <= inp_2
+
+; VERILOG-LABEL:   module implicitTrunc(
+; VERILOG-NEXT :   input        inp_1,
+; VERILOG-NEXT :   input  [4:0] inp_2,
+; VERILOG-NEXT :   output [2:0] tmp9);
+; VERILOG-NEXT :   assign out1 = $signed(inp_2) << inp_1;
+; VERILOG-NEXT :   assign out2 = inp_2;
+; VERILOG-NEXT : endmodule
```

Comment:
This check of the `EmitVerilog` path looks good. Can you also add a test of the `-lower-to-rtl` path?